src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1016,6 +1016,68 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         _SetFontSize(newSize);
     }
 
+    // Method Description:
+    // - Moves the selection anchor in use. Used for Keyboard Selection.
+    // Arguments:
+    // - dir: The direction to move the selection anchor in
+    // - expansionMode: The movement mode in use for movement
+    // Return Value:
+    // - true, if the action was handled. false, otherwise.
+    bool TermControl::MoveSelectionAnchor(Settings::Direction dir, Settings::SelectionExpansionMode mode)
+    {
+        // no selection --> not valid
+        if (_terminal == nullptr || !_terminal->IsSelectionActive())
+        {
+            return false;
+        }
+
+        // translate Settings::Direction to Terminal::Direction
+        ::Microsoft::Terminal::Core::Terminal::Direction selectionDir;

```

Comment:
Yea this could be avoided by just having the `Terminal` use `Settings::Direction` instead of manually duplicating it.

Reply:
I'm unresolving this - Since the Terminal Core needs to be able to read from a `ICoreSettings` (which is declared in `..Terminal.Settings.winmd`), then it can definitely just accept parameter types on it's methods that are _also_ defined in `.Terminal.Settings.winmd` (IMO).